plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```diff
@@ -73,6 +78,21 @@ public function trigger( $order_id, $order = false ) {
 			$this->restore_locale();
 		}
 
+		/**
+		 * Trigger the sending of this email when requested via the REST API.
+		 *
+		 * @param int    $order_id    The order ID.
+		 * @param string $template_id The email template ID.
+		 */
+		public function maybe_trigger_from_api( $order_id, $template_id ) {
```

Comment:
Curious about this one: is it applicable (or already) in the second template we are touching in this PR?

Reply:
The triggers for these 2 email classes are different:

- POS completed order email: triggered manually via the API in the Woo mobile apps `/wc/v3/orders/$orderId/actions/send_email`
- POS refunded order email: triggered automatically from [order refund notification actions](https://github.com/woocommerce/woocommerce/blob/d8e4eed76569f66e314197892f488d19216cd054/plugins/woocommerce/includes/emails/class-wc-email-customer-refunded-order.php#L56-L58), which will be implemented separately in WOOMOB-405

Therefore, the support for API email trigger isn't needed for the POS refunded order email class.

Reply:
Gotcha. Thanks for explaining.

I think it makes more sense to move this check into `\WC_Email_Customer_POS_Completed_Order::trigger` method (it has some similar bits, so it looks like we need to refactor it a bit).

Reply:
Good point, the original `trigger` in the POS completed order email class can be updated for the API order send email action as this template is meant to be sent manually. In https://github.com/woocommerce/woocommerce/pull/57573/commits/c11768ba548ea87df86dd9089300972e417c2f23 I updated this email class to be manually sent, and just have one `trigger` function triggered by `woocommerce_rest_order_actions_email_send`.